extensions/pyRevitTools.extension/pyRevit.tab/Drawing Set.panel/views.stack/Views.pulldown/Move Viewport Label.pushbutton/script.py

```diff
@@ -0,0 +1,45 @@
+# encoding: utf-8
+# from https://discourse.pyrevitlabs.io/t/i-developed-a-tool-need-help-to-get-it-into-the-new-release-of-pyrevit/7639/3
+
+from pyrevit import DB, UI, script, HOST_APP, revit
+
+doc = HOST_APP.doc
+uidoc = HOST_APP.uidoc
+script.get_output().close_others()
+
+
+def move_viewport_label(viewport, point):
+    viewminpoint = viewport.GetBoxOutline().MinimumPoint
+    new_label_location = point - viewminpoint
+    viewport.LabelOffset = new_label_location
+
+
+def single_point_selection():
```

Comment:
since you translated it to python, why not use `pyrevit.revit.selection.pick_point()` directly in the main script?